_data/harnesses/kafka-consumer-application/confluent.yml

```diff
@@ -0,0 +1,162 @@
+          render:
+            skip: true
+
+    - title: Compile and run the Kafka Consumer program
+      content:
+        - action: execute
+          file: tutorial-steps/dev/build-uberjar.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/build-uberjar.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/run-dev-app.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/run-dev-app.adoc
+
+    - title: Produce sample data to the input topic
+      content:
+        - action: skip
+          render:
+            file: tutorials/kafka-consumer-application/confluent/markup/dev/ccloud-run-producer.adoc
+
+    - title: Inspect the consumed records
+      content:
+        - action: execute
+          file: tutorial-steps/dev/print-consumer-file-results.sh
+          stdout: tutorial-steps/dev/outputs/actual-output.txt
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/print-consumer-file-results.adoc
+
+
+test:
+  steps:
+    - title: Create a test configuration file
+      content:
+        - action: make_file
+          file: configuration/test.properties
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/test/make-test-file.adoc
+
+    - title: Write a test for the consumer application
+      content:
+        - action: execute
+          file: tutorial-steps/test/make-test-dir.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/test/make-test-dir.adoc
+
+        - action: make_file
+          file: src/test/java/io/confluent/developer/KafkaConsumerApplicationTest.java
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/test/make-consumer-applicaion-test.adoc
+
+    - title: Write a test for the records ConsumerRecordsHandler
+      content:
+        - action: make_file
+          file: src/test/java/io/confluent/developer/FileWritingRecordsHandlerTest.java
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/test/make-consumer-records-handler-test.adoc
+
+
+    - title: Invoke the tests
+      content:
+        - action: execute
+          file: tutorial-steps/test/invoke-tests.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/test/invoke-tests.adoc
+
+prod:
+  steps:
+    - title: Create a production configuration file
+      content:
+        - action: make_file
+          file: configuration/prod.properties
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/prod/make-prod-file.adoc
+
+    - title: Build a Docker image
+      content:
+        - action: execute
+          file: tutorial-steps/prod/build-image.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/prod/build-image.adoc
+
+    - title: Launch the container
```

Comment:
There's no Docker container any more so this should be removed.

Reply:
Actually the step before this --> `gradle jibDockerBuild --image=io.confluent.developer/kafka-consumer-application-join:0.0.1` creates a docker image for you to run, so I believe this step is still valid.

Reply:
THANK YOU!  I forgot the app itself is Docker